File: Source/Fantasy.Jobs/TrackStatusService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Fantasy.ServiceModel;
using Fantasy.Tracking;

namespace Fantasy.Jobs
{
    public class TrackStatusService : AbstractService, IStatusBarService
    {

        private ITrackProvider _track;

        public override void InitializeService()
        {
            _track = this.Site.GetService<ITrackProvider>();
            base.InitializeService();
        }

        #region IStatusBarService Members

        public void SetStatus(string status)
        {
            _track["status"] = status;
        }

        #endregion
    }
}

File: AutoConverted/Fantasy.Jobs/TrackStatusService.java
package Fantasy.Jobs;

import Fantasy.ServiceModel.*;
import Fantasy.Tracking.*;

public class TrackStatusService extends AbstractService implements IStatusBarService
{

	private ITrackProvider _track;

	@Override
	public void InitializeService()
	{
		_track = this.Site.<ITrackProvider>GetService();
		super.InitializeService();
	}

//C# TO JAVA CONVERTER TODO TASK: There is no preprocessor in Java:
		///#region IStatusBarService Members

	public final void SetStatus(String status)
	{
		_track["status"] = status;
	}

//C# TO JAVA CONVERTER TODO TASK: There is no preprocessor in Java:
		///#endregion
}